sites/docs/src/pages/_ViewConfigEditor.js

```diff
@@ -157,6 +181,34 @@ export default function ViewConfigEditor(props) {
           <button type="button" onClick={tryExample}>Try an example</button>&nbsp;
           {showReset && <button type="button" onClick={resetEditor}>Reset the editor</button>}
         </p>
+
+        <div className={styles.viewConfigInputs}>
+          <div className={styles.viewConfigInputUrlOrFile}>
+            <p className={styles.viewConfigInputUrlOrFileText}>
+              Or paste the URLs of your datasets and a&nbsp;
+              <a href={defaultViewConfigDocsUrl}>default view config</a>
+                  &nbsp; will be displayed in the editor below.
+            </p>
+            <div className={styles.generateConfigInputUrl}>
+              <input
+                type="text"
+                className={styles.viewConfigUrlInput}
+                placeholder="Put one or more URLs to your datasets in here, separating each URL with a semicolon."
```

Comment:
i.e., re: my above comment about file extensions, we could add an "override" button to allow for people to open non-standard-ly named files.

Reply:
Related to https://github.com/vitessce/vitessce/pull/1455#discussion_r1152943671 - i.e., allow people to specify the file directly without having to have this inferred from the suffix

Reply:
I think for a first pass at addressing https://github.com/vitessce/vitessce/issues/1406 relying on the file extension is a good compromise. However in the long term, I agree that there should be a way to override this.

In https://github.com/vitessce/vitessce/issues/1406, I proposed the input to the auto-config function could be "one or more file URLs plus some hints" - I think the hints support could be added in a subsequent PR, with one potential use case to be overriding the file extension detection.

Alternatively, I also don't really have an issue with continuing to rely on file extensions for auto-configuration, as there is always the "fallback" to manually write the config using JSON or one of the object-oriented config APIs.

Reply:
@ilan-gold and @keller-mark,
Do we all agree that relying on the file extension for issue #1406 is ok? Should I create a new issue to remove the relying on the file extension, so we can tackle that in the future?

On the topic of adding hints, should I tackle it in a subsequent PR, still part of issue #1406 or do we want to create a new issue for this and tackle it in the future?

Reply:
Discussed here: https://hms-dbmi.slack.com/archives/GG50HSQ4E/p1680602387564519 I will create a new issue and we can tackle it as part of a new issue.